src/Controller/V2/Document/DocumentRoute.ts

```diff
@@ -0,0 +1,132 @@
+/*!
+ * © 2023 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { celebrate } from 'celebrate'
+import { NextFunction, Request, Response, Router } from 'express'
+import { StatusCodes } from 'http-status-codes'
+
+import { AuthStrategy } from '../../../Auth/Passport/AuthStrategy'
+import { BaseRoute } from '../../BaseRoute'
+import { DocumentController } from './DocumentController'
+import {
+  createDocumentSchema,
+  deleteDocumentSchema,
+  getDocumentSchema,
+  updateDocumentSchema,
+} from './DocumentSchema'
+
+export class DocumentRoute extends BaseRoute {
+  private documentController = new DocumentController()
+
+  private get basePath(): string {
+    return '/doc'
+  }
+
+  public create(router: Router): void {
+    router.post(
+      `${this.basePath}/:projectID/manuscript/:manuscriptID`,
+      celebrate(createDocumentSchema),
+      AuthStrategy.JsonHeadersValidation,
+      AuthStrategy.JWTAuth,
+      (req: Request, res: Response, next: NextFunction) => {
+        return this.runWithErrorHandling(async () => {
+          await this.createDocument(req, res)
+        }, next)
+      }
+    )
```

Comment:
I just realized we're receiving manuscriptID twice, once in the params and once in the body of the request. And we're not using the one in the params at all. 
Should I replace it with `basePath/:projectID` or the same route but remove manuscript_model_id from the body?
@hhusban @mnatsheh

Reply:
I think it's better to use the param. If it's easy to update the FE to not add the body part, let's do it, otherwise we can just ignore in the backend until it's removed from FE.